src/main/webapp/app/exercises/programming/manage/services/programming-exercise.service.ts

```diff
@@ -103,6 +104,7 @@ export class ProgrammingExerciseService {
      */
     importExercise(adaptedSourceProgrammingExercise: ProgrammingExercise, recreateBuildPlans: boolean, updateTemplate: boolean): Observable<EntityResponseType> {
         const options = createRequestOption({ recreateBuildPlans, updateTemplate });
+        this.exerciseService.setBonusPointsConstrainedByIncludedInOverallScore(adaptedSourceProgrammingExercise);
```

Comment:
the bonus points from the exercise are set in the call but the result of the call is not assigned. Is this on purpose?

Reply:
Good catch! Fixed it in 52a75a0c250171f2e3baed9283f67126df2c763e